pkg/distributor/distributor.go

```diff
@@ -67,12 +78,12 @@ var (
 		Namespace: "cortex",
 		Name:      "distributor_ingester_appends_total",
 		Help:      "The total number of batch appends sent to ingesters.",
-	}, []string{"ingester"})
+	}, []string{"ingester", "type"})
```

Comment:
I tend to favour this approach. Using a new metric can make the change go unnoticed and when investigating things our users might not be aware of this change (although a user should always read the changelog). 

Also, creating a new metric suggest that it would be something like `distributor_ingester_metadata_appends_total`, which makes `distributor_ingester_appends_total` a bit more confusing w/o the word sample in there (since we wouldn't want to change what we have). 

I'm happy to change it if you disagree though.

Reply:
> when investigating things our users might not be aware of this change (although a user should always read the changelog).

On the other hand, introducing new label may break existing graphs / alerts people have :) Let's make sure it's mentioned in the changelog.

Reply:
I currently have `[CHANGE] The metrics `cortex_distributor_ingester_appends_total` and `distributor_ingester_append_failures_total` now includes a `type` label to differentiate between `samples` and `metadata`.`. is there anything else you think I should add?

Reply:
Looks good, thanks!